lib/modules/dosomething/dosomething_fact_page/dosomething_fact_page.module

```diff
@@ -90,22 +89,29 @@ function dosomething_fact_page_preprocess_node(&$vars) {
 
     foreach ($template_vars as $key => $labels) {
       foreach ($labels as $label) {
-        $field = "field_{$label}";
-          if (isset($content[$field])) {
+        $field = $vars["field_{$label}"];
+
+        if (isset($field) && !empty($field)) {
+          if (array_key_exists('en', $field)) {
+            $field = $field['en'];
```

Comment:
to protect against the #future could we foreach around the [`language_list`](https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/language_list/7)